mandatory/1-fix-functions.js

```diff
@@ -21,9 +19,9 @@ function getMood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
 
-  if (isBigEnough) {
+
+  if (num>10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
```

Comment:
Hi Baki
what you did here is awesome well done
and I add for you this you can make it shorter like that 

function greaterThan10(num) {
  return num > 10 ? "num is greater than 10" : "num is not big enough";
}

This is my review for this week 
Thanks